src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,33 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
```

Comment:
Ensure that the logger is configured with both File and Console appenders as specified in the task description.